spk/adminer/Makefile

```diff
@@ -21,22 +20,37 @@ INSTALLER_SCRIPT = src/installer.sh
 SSS_SCRIPT = src/dsm-control.sh
 WIZARDS_DIR = src/wizard/
 
+include ../../mk/spksrc.common.mk
+
+##
+## This package can be built for DSM <6.0, DSM >=6.0;<7.0 and DSM >=7.0
+## To bild all versions call:
+## make && make TCVERSION=6.0 && make TCVERSION=7.0 
+##
+
+ifeq ($(call version_ge, ${TCVERSION}, 6.0),1)
 SPK_DEPENDS = "WebStation>=2.1.0:PHP7.4>=7.4.0:Apache2.4>=2.4.0"
+CONF_DIR = src/conf
+APACHE_VERSION = 2.4
+else
+APACHE_VERSION = 2.2
+endif
```

Comment:
Doesn't `SPK_DEPENDS` apply for the else statement for `Apache2.2`? or was it not implemented in DSM5?

Reply:
DSM5 has apache integrated into DSM and not as installable packages.

Reply:
Ok Thanks,